src/styles/index.scss

```diff
@@ -1,3 +1,66 @@
 body {
   margin: 0;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  min-height: 100vh;
+}
+
+.calendar {
+  display: flex;
+  flex-wrap: wrap;
+  width: 706px;
+  gap: 1px;
+  padding: 10px;
+
+  &__day {
+    display: flex;
+    box-sizing: border-box;
+    height: 100px;
+    width: 100px;
+    background-color: #eee;
+    border: 1px solid black;
+    justify-content: center;
+    align-items: center;
+  }
+
+  @for $number from 1 through 31 {
+    & :nth-child(#{$number})::before {
+      content: '#{$number}';
+      font-family: Arial, Helvetica, sans-serif;
+      font-size: 30px;
```

Comment:
```suggestion

```
It's better to use a regular styling in the regular class instead of generating it for every element in the cycle